src/HtmlGenerator/Program.cs

```diff
@@ -72,8 +72,8 @@ private static void Main(string[] args)
                 Paths.SolutionDestinationFolder = Path.Combine(Microsoft.SourceBrowser.Common.Paths.BaseAppFolder, "Index");
             }
 
-            Log.ErrorLogFilePath = Path.Combine(Paths.SolutionDestinationFolder, Log.ErrorLogFile);
-            Log.MessageLogFilePath = Path.Combine(Paths.SolutionDestinationFolder, Log.MessageLogFile);
+            Log.ErrorLogFilePath = Path.GetFullPath(Path.Combine(Paths.SolutionDestinationFolder, Log.ErrorLogFile));

```

Comment:
I think it's better to ensure that Paths.SolutionDestinationFolder is rooted full path, instead of here. SolutionDestinationFolder is central to the entire app and I think having .. inside it would be bad in various other places.